docs_source_files/content/webdriver/page_loading_strategy.de.md

```diff
@@ -9,6 +9,17 @@ it follows the _normal_ pageLoadStrategy.
 It is always recommended to stop downloading additional 
 resources (like images, css, js) when the page loading takes lot of time.
 
+`document.readyState` property of a document describes the loading state of current document.
+By default, webdriver will hold off on responding to a `driver.get()` (or) `driver.navigate().to()` 
+call until the document ready state is `complete`
+
+### Note: 
+In SPA (Single Page Applications) as once they reach the document ready state of complete, 
```

Comment:
This paragraph is not clear, I don't understand what is trying to say.

Could you maybe describe what you want to achieve and we could work on it together? If you have any links where the text is based on, that'd be helpful.

Reply:
Hi @diemol, Updated content. Can you please review it.